### ISSUE TITLE
Feature request: Require an option field to be set

### ISSUE THREAD

**Issue by marium0505** [reduxframework/redux-framework#3608]
Make it possible to require an option to be set. This should apply to any field that contains more than one option (for instance the Button Set, Checkbox and Select field). 

What I mean with this is that, for instance when a user unchecks all the options in a checkbox field, one (or more) option will be automatically set (and/or the settings wont be saved). For instance can the field be set to the default values (or something else). 

To summarize: There should be a new argument that makes it possible to requires something to be set (and if nothing is set, either force something or output an error when trying to save the settings).

**Comment by kprovance** [comment]
Handled in slack.  Thanks!
